c_binding/include/libstoragemgmt/libstoragemgmt_local_disk.h

```diff
@@ -73,6 +73,37 @@ int LSM_DLL_EXPORT lsm_local_disk_vpd83_get(const char *sd_path,
                                             char **vpd83,
                                             lsm_error **lsm_err);
 
+/**
+ * Query the disk rotation speed - revolutions per minute(RPM) of given SCSI
+ * disk path.
```

Comment:
Probably should remove `SCSI` as the device could be SATA too.  Should this be more generic term as storage doesn't have to be disk, NVMe etc.?